src/data/metadata.csv

```diff
@@ -10614,6 +10614,8 @@ ID_GO,ID,Indonesia,GO,Gorontalo,,,,,,
 ID_JA,ID,Indonesia,JA,Jambi,,,,,,
 ID_JB,ID,Indonesia,JB,West Java,,,,,Jawa Barat,
 ID_JI,ID,Indonesia,JI,East Java,,,,,Jawa Timur,
+ID_JI_35.04,ID,Indonesia,JI,East Java,35.04,Tulungagung
+ID_JI_35.23,ID,Indonesia,JI,East Java,35.23,Tuban
```

Comment:
We normally don't use `.` as part of the location keys. I'd suggest removing the period altogether and use (zero padded) integers i.e. `3504` and `3523`.

Reply:
done